.moban.cd/moban.yml

```diff
@@ -15,9 +15,10 @@ keywords:
   - jinja2
   - moban
 dependencies:
-  - ruamel.yaml
+  - ruamel.yaml==0.15.86
```

Comment:
was there some feature which needed .86 ?

https://build.opensuse.org/package/show/devel:languages:python/python-ruamel.yaml is on .80

I'll try updating it, but that will take longer.

Reply:
just wanted to nominate a version instead of any.

Reply:
what's the one available on OpenSuse? without considering distros, any working version would be OK.

Reply:
build.opensuse.org/package/show/devel:languages:python/python-ruamel.yaml is on .80 ; that is good enough for now.